pages/jobs.tsx

```diff
@@ -1,35 +1,215 @@
+import { useState } from 'react';
+import Select from 'components/Select';
+
+type JobEntryProps = {
+    job: JobSummary;
+};
+
+const JobEntry = ({ job }: JobEntryProps) => (
+    <>
+        <Link href={'/jobs/' + job.slug}>
+            <a className={styles.jobTitle}>{job.title}</a>
+        </Link>
+        <p className={styles.jobDescription}>{job.description}</p>
+        <div className={styles.jobMetadataItems}>
+            <div className={styles.jobMetadata}>
+                <LocationIcon
+                    width={22}
+                    height={23}
+                    className={styles.jobMetadataIcon}
+                />{' '}
+                {job.department}
+            </div>
+            <div className={styles.jobMetadata}>
+                <LocationIcon
+                    width={22}
+                    height={23}
+                    className={styles.jobMetadataIcon}
+                />{' '}
+                <div>{job.city}</div>
+            </div>
+        </div>
+    </>
+);
+
+type CTAProps = {
+    title: string;
+    children: React.ReactNode;
+};
+
+const CTA = ({ title, children }: CTAProps) => (
+    <>
+        <h2 className={styles.ctaTitle}>{title}</h2>
+        <div className={styles.ctaDescription}>{children}</div>
+        <SwishButton jobs={0} url="mailto:recruitment@torchbox.com">
+            Send us your CV
+        </SwishButton>
+    </>
+);
+
+type JobsPageProps = {
+    preview: boolean;
+    jobs: JobSummary[];
+    content: Jobs;
+};
+
+const Jobs: NextPage<JobsPageProps> = ({ preview, jobs, content }) => {
+    const jobsAvailable = jobs && jobs.length > 0;
+
+    const [location, setLocation] = useState('Locations');
+    const [department, setDepartment] = useState('Departments');
+
+    const locations = jobs.reduce(
+        (acc: string[], job: JobSummary) => {
+            if (!acc.includes(job.city)) {
+                acc.push(job.city);
+            }
+            return acc;
+        },
+        ['Locations'],
+    );
+    const departments = jobs.reduce(
+        (acc: string[], job: JobSummary) => {
+            if (!acc.includes(job.department)) {
+                acc.push(job.department);
+            }
+            return acc;
+        },
+        ['Departments'],
+    );
+
+    if (location !== 'Locations') {
+        jobs = jobs.filter((job) => job.city === location);
+    }
+    if (department !== 'Departments') {
+        jobs = jobs.filter((job) => job.department === department);
+    }
+
+    const jobEntries = jobs.map((job: JobSummary, index: number) => (
+        <JobEntry job={job} key={index} />
+    ));
+
+    const resetFilterKeyDownHandler = (
+        event: React.KeyboardEvent<HTMLParagraphElement>,
+    ) => {
+        if (!['Tab', 'Shift', 'ArrowDown', 'ArrowUp'].includes(event.key)) {
+            resetFilter();
+        }
+    };
+
+    const resetFilter = () => {
+        setLocation('Locations');
```

Comment:
**Suggestion**: since you're using the string 'Locations' and 'Departments' multiple times, it would be nice to use a variable, i.e. `ALL_LOCATIONS = 'Locations'`